app/main/templates/index.html

```diff
@@ -52,7 +52,7 @@ <h5 class="card-title">Uniformity</h5>
         <h6 class="card-subtitle mb-2 text-muted"></h6>
     <p class="card-text"></p>
       <div class="card-footer">
-       <a href="https://readthedocs.org/projects/hazen/" class="btn mr-2"><i class="fas fa-link"></i>Docs</a>
+       <a href="https://hazen.readthedocs.io/en/fix-readthedocs/tasks.html#uniformity" class="btn mr-2"><i class="fas fa-link"></i>Docs</a>
```

Comment:
@Lucrezia-Cester Looks great, but let's update the links to point to the `latest` version:
``` suggestion
       <a href="https://hazen.readthedocs.io/en/latest/tasks.html#uniformity" class="btn mr-2"><i class="fas fa-link"></i>Docs</a>
```

@hshuaib90 Could we please remove the `fix-readthedocs` and `add-rtd-yaml` branches from being tracked in the Read The Docs builds e.g, it should only track `main` and `release` branches?
![image](https://user-images.githubusercontent.com/67117138/214030048-11071249-ee7e-4405-930f-e02e59309d8e.png)